packages/home-service/.env.example

```diff
@@ -19,3 +19,6 @@ COOKIE_PARSER=HOME_SECRET
 
 ## Authorization Token
 # AUTH_TOKEN=
+
+## User-Service
+USER_SERVICE_SERVICE_API=
```

Comment:
Should be `USER_SERVICE_SERVICE_HOST`.
And for the port, use `USER_SERVICE_SERVICE_PORT`.

Also, it will only give you the IP address of the user-service. You'll have to concat `/graphql` in the code.

Reply:
Changed it in the env.example and also the fetch